libs/tools/export/vault-export/vault-export-core/src/services/org-vault-export.service.ts

```
@@ -295,16 +300,21 @@ export class OrganizationVaultExportService
         !this.restrictedItemTypesService.isCipherRestricted(f, restrictions),
     );
 
-    return this.BuildEncryptedExport(organizationId, encCollections, encCiphers);
+    return this.BuildEncryptedExport(activeUserId, organizationId, encCollections, encCiphers);
   }
 
   private async BuildEncryptedExport(
+    activeUserId: UserId,
     organizationId: OrganizationId,
     collections: Collection[],
     ciphers: Cipher[],
   ): Promise<string> {
-    const orgKey = await this.keyService.getOrgKey(organizationId);
-    const encKeyValidation = await this.encryptService.encryptString(Utils.newGuid(), orgKey);
+    const orgKeys = await firstValueFrom(this.keyService.orgKeys$(activeUserId));
+    let keyForEncryption: SymmetricCryptoKey = orgKeys?.[organizationId];
+    if (keyForEncryption == null) {
+      keyForEncryption = await firstValueFrom(this.keyService.userKey$(activeUserId));
```

I don't think we should have a UserKey fallback, because this is Org export and not Individual vault export.
In those cases i would recommend to throw an error, since is would be abnormal for org key to be absent.
Note: this is not the same as for the `libs/importer/src/importers/bitwarden/bitwarden-json-importer.ts` change, which can handle import of both Org and User vault, hence why the UserKey key fallback (for Individual) make sense. Although i wonder if this should not he handled differently in that code, creating clear separation between Org and User import, throwing error when the decryption key is not found, without any fallback.